src/sentry/snuba/discover.py

```diff
@@ -322,6 +326,10 @@ def timeseries_query(
         Dataset.Discover,
         Dataset.Transactions,
     ], "A dataset is required to query discover"
+
+    if len(params) == 0 and snuba_params is not None:
+        params = snuba_params.filter_params
+
```

Comment:
This is a pattern i reuse repeatedly, its so in future PRs i can pass `params={}` and just pass snuba_params, I'll then follow up with another PR where i'll start deleting `params={}` everywhere